corehq/apps/case_search/templates/case_search/bootstrap5/case_search.html

```diff
@@ -82,17 +90,21 @@ <h3>{% trans "Search Parameters" %}</h3>
                 <span class="btn btn-outline-primary" id="add-input" data-bind="click: addParameter"><i class="fa fa-plus"></i> {% trans "Add input" %}</span>
               </div>
             </div>
-            <div class="card card-default">
+            <div class="card card-default mb-3">
```

Comment:
do we have a guide for adding margin like this? Or did you just add it make it look better?

Reply:
Hi Martin, I just add it to make it look better. I think the `panel` used to have margin or padding, but `card` doesn't have anything. Without adding it, the three cards will have no spacing between them at all.